crates/genesis/src/lib.rs

```diff
@@ -423,6 +423,10 @@ pub struct ChainConfig {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub clique: Option<CliqueConfig>,
 
+    /// Parlia parameters.
```

Comment:
can we add an url to to a spec if that exists?

Reply:
thanks a lot @mattsse 
The introduction and code specification are added.